### PR TITLE
Change names of BaseFieldVec, SecureFieldVec, SecureColumn to BaseColumn, SecureColumn, SecureColumnByCoords respectively.

### PR DESCRIPTION
**Pull request by Alon-Ti**
BaseColumn, SecureColumn, SecureColumnByCoords respectively.

- - -
This change is [Reviewable](https://reviewable.io/reviews/starkware-libs/stwo/748)

### CODE REVIEW

**Comment by Alon-Ti** [comment]
* **#750** <a href="https://app.graphite.dev/github/pr/starkware-libs/stwo/750?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#749** <a href="https://app.graphite.dev/github/pr/starkware-libs/stwo/749?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#748** <a href="https://app.graphite.dev/github/pr/starkware-libs/stwo/748?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Alon-Ti and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by shaharsamocha7 — CHANGES_REQUESTED** [review]
> Reviewed 26 of 26 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/stwo/748)* status: all files reviewed, 4 unresolved discussions (waiting on @Alon-Ti and @spapinistarkware)

___
*[`crates/prover/src/core/backend/simd/column.rs` line 29 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2OfhtBEL_q120zpwh3:-O2OfhtBEL_q120zpwh4:bbfn7s0) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/backend/simd/column.rs#L29)):*
> ```XML
> }
> 
> /// A efficient structure for storing and operating on a arbitrary number of [`BaseField`] values.
> ```

Can you fix?



_Suggestion:_
```XML
An efficient structure for storing and operating on a arbitrary number of [`BaseField`] values.
```
___
*[`crates/prover/src/core/backend/simd/column.rs` line 102 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2OfnVy5RIZX_wURrFi:-O2OfnVy5RIZX_wURrFj:bb7od0r) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/backend/simd/column.rs#L102)):*
> ```XML
> }
> 
> /// A efficient structure for storing and operating on a arbitrary number of [`SecureField`] values.
> ```

same



_Suggestion:_
```XML
An efficient structure for storing and operating on a arbitrary number of [`SecureField`] v
```
___
*[`crates/prover/src/core/fields/secure_column.rs` line 11 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2Og5z30w8TnOKYh_TC:-O2Og5z30w8TnOKYh_TD:bfilhoj) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/fields/secure_column.rs#L11)):*
> ```XML
> 
> /// An array of `SECURE_EXTENSION_DEGREE` base field columns, that represents a column of secure
> /// field elements.
> ```

Can you rephrase this?



_Code quote:_
```XML
/// An array of `SECURE_EXTENSION_DEGREE` base field columns, that represents a column of secure
/// field elements.
```
___
*[`crates/prover/src/core/fields/secure_column.rs` line 24 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2OgPOIEWWbWkElmBjN:-O2OgPOIEWWbWkElmBjO:byyjtpq) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/fields/secure_column.rs#L24)):*
> ```XML
>     }
> 
>     // TODO(spapini): Remove when we no longer use CircleEvaluation<SecureField>.
> ```

Can we remove this function now?



_Code quote:_
```XML
    // TODO(spapini): Remove when we no longer use CircleEvaluation<SecureField>.
```

**Review by Alon-Ti — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/stwo/748)* status: 24 of 26 files reviewed, 4 unresolved discussions (waiting on @shaharsamocha7 and @spapinistarkware)

___
*[`crates/prover/src/core/backend/simd/column.rs` line 29 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2OfhtBEL_q120zpwh3:-O2OoX1K7wAXNAQTZWZd:b-896fix) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/backend/simd/column.rs#L29)):*
<details><summary><i>Previously, shaharsamocha7 wrote…</i></summary><blockquote>

Can you fix?
</blockquote></details>

Done.
___
*[`crates/prover/src/core/backend/simd/column.rs` line 102 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2OfnVy5RIZX_wURrFi:-O2OoX_fDdccGiOV-I1w:b-896fix) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/backend/simd/column.rs#L102)):*
<details><summary><i>Previously, shaharsamocha7 wrote…</i></summary><blockquote>

same
</blockquote></details>

Done.
___
*[`crates/prover/src/core/fields/secure_column.rs` line 11 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2Og5z30w8TnOKYh_TC:-O2OpNBd8ywJWt-E_AYG:b-c6ityx) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/fields/secure_column.rs#L11)):*
<details><summary><i>Previously, shaharsamocha7 wrote…</i></summary><blockquote>

Can you rephrase this?
</blockquote></details>

I can try, do you have anything specific in mind?
___
*[`crates/prover/src/core/fields/secure_column.rs` line 24 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2OgPOIEWWbWkElmBjN:-O2Oq0ghDrx1c1TC6rZl:bxuta8w) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/fields/secure_column.rs#L24)):*
<details><summary><i>Previously, shaharsamocha7 wrote…</i></summary><blockquote>

Can we remove this function now?
</blockquote></details>

Not trivially, this is in use in `crates/prover/src/core/backend/simd/quotients.rs:314:10` for example. Maybe the comment is no longer correct though?

**Review by Alon-Ti — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/stwo/748)* status: 24 of 26 files reviewed, 4 unresolved discussions (waiting on @shaharsamocha7 and @spapinistarkware)

___
*[`crates/prover/src/core/fields/secure_column.rs` line 11 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2Og5z30w8TnOKYh_TC:-O2OrX9A3D3VRXSCCmc9:b-896fix) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/fields/secure_column.rs#L11)):*
<details><summary><i>Previously, Alon-Ti wrote…</i></summary><blockquote>

I can try, do you have anything specific in mind?
</blockquote></details>

Done.

**Review by shaharsamocha7 — APPROVED** [review]
:lgtm:

> Reviewed 2 of 2 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/starkware-libs/stwo/748#-:-O2Ov3Rj14tWFm5KudHf:bnfp4nl)* status: all files reviewed, 1 unresolved discussion (waiting on @spapinistarkware)

___
*[`crates/prover/src/core/fields/secure_column.rs` line 24 at r1](https://reviewable.io/reviews/starkware-libs/stwo/748#-O2OgPOIEWWbWkElmBjN:-O2Ov-_cCrVSaCvxIcdl:bqf7n20) ([raw file](https://github.com/starkware-libs/stwo/blob/246741e2dd5b086dc071642941fd3719c4173ebb/crates/prover/src/core/fields/secure_column.rs#L24)):*
<details><summary><i>Previously, Alon-Ti wrote…</i></summary><blockquote>

Not trivially, this is in use in `crates/prover/src/core/backend/simd/quotients.rs:314:10` for example. Maybe the comment is no longer correct though?
</blockquote></details>

@spapinistarkware can we remove this TODO?

**Comment by Alon-Ti** [comment]
### Merge activity

* **Jul 23, 7:29 AM EDT**: @Alon-Ti started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/starkware-libs/stwo/748).
